src/core/Akka.Remote/Serialization/ExceptionSupport.cs

```diff
@@ -104,22 +113,25 @@ public Exception ExceptionFromProtoNet(Proto.Msg.ExceptionData proto)
 
             Type exceptionType = Type.GetType(proto.TypeName);
 
-            var serializationInfo = new SerializationInfo(exceptionType, DefaultFormatterConverter);
+            var serializationInfo = new SerializationInfo(exceptionType, _defaultFormatterConverter);
 
             serializationInfo.AddValue("ClassName", proto.TypeName);
-            serializationInfo.AddValue("Message", proto.Message);
-            serializationInfo.AddValue("StackTraceString", proto.StackTrace);
-            serializationInfo.AddValue("Source", proto.Source);
+            serializationInfo.AddValue("Message", ValueOrNull(proto.Message));
+            serializationInfo.AddValue("StackTraceString", ValueOrNull(proto.StackTrace));
+            serializationInfo.AddValue("Source", ValueOrNull(proto.Source));
             serializationInfo.AddValue("InnerException", ExceptionFromProto(proto.InnerException));
             serializationInfo.AddValue("HelpURL", string.Empty);
-            serializationInfo.AddValue("RemoteStackTraceString", string.Empty);
+            serializationInfo.AddValue("RemoteStackTraceString", null);
             serializationInfo.AddValue("RemoteStackIndex", 0);
-            serializationInfo.AddValue("ExceptionMethod", string.Empty);
+            serializationInfo.AddValue("ExceptionMethod", null);
             serializationInfo.AddValue("HResult", int.MinValue);
 
             foreach (var field in proto.CustomFields)
             {
-                serializationInfo.AddValue(field.Key, _wrappedPayloadSupport.PayloadFrom(field.Value));
+                var payload = _wrappedPayloadSupport.PayloadFrom(field.Value);
+                if (payload is ExceptionData exception)
+                    payload = ExceptionFromProto(exception);
```

Comment:
Old code would fail if it encounters an `Exception` property inside an `Exception`, have to make sure that the serializer recurse through these